docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -14,39 +15,104 @@
 
 \begin{document}
 
+\maketitle
+
+\newpage
+
 \begin{table}[hp]
 \caption{Revision History} \label{TblRevisionHistory}
 \begin{tabularx}{\textwidth}{llX}
 \toprule
 \textbf{Date} & \textbf{Developer(s)} & \textbf{Change}\\
 \midrule
-Date1 & Name(s) & Description of changes\\
-Date2 & Name(s) & Description of changes\\
-... & ... & ...\\
+Sept 21/22 & Talha & Updating Workflow Plan\\
+Sept 24/22 & Talha & Updating Sections 1-3\\
```

Comment:
I changed the formatting around but if its gonna cost marks cause its different then I'll change back 👀